package.json

```diff
@@ -12,8 +12,9 @@
     "lint:fix": "prettier --write 'src/**/*.{js,json,jsx}'"
   },
   "dependencies": {
+    "ajv": "^6.10.0",
     "firebase": "^5.11.1",
-    "firebase-admin": "~6.0.0",
+    "firebase-admin": "^7.4.0",
```

Comment:
Seems like we should also have a new lock file.

Reply:
Yup - it didn't update because someone(...?) put the lock file in the .gitignore. ;)

Reply:
Should I remove it from .gitignore?

Reply:
To avoid confusion the lock file from npm was ignored in favor or `yarn.lock`.

Reply:
Only one should be committed. Either from yarn or npm.

Reply:
We are using yarn in CI so we should probably update the `yarn.lock` unless there is a plan to switch CI to use npm.

Reply:
Ok, does that mean I should do `yarn install` before each build? I'm trying to figure out how to add this to explanations in the README.

Reply:
No, the `yarn.lock` only changes when you change the dependencies.

Reply:
If it is confusing you might add that dependencies are managed using yarn to the README.md. This should be evident by the `yarn.lock` and yarn commands already in the README.md but it won't hurt to be more explicit.